hugegraph-ml/src/hugegraph_ml/data/hugegraph2dgl.py

```diff
@@ -0,0 +1,84 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import dgl
+import torch
+from pyhugegraph.api.gremlin import GremlinManager
+from pyhugegraph.client import PyHugeClient
+
+
+class HugeGraph2DGL:
+    def __init__(
+            self,
+            ip='127.0.0.1',
+            port="8080",
+            graph='hugegraph',
+            user='admin',
+            pwd='xxx',
```

Comment:
better to use `""` or `admin`?

Reply:
I'm using `"admin"` here because the default config in https://github.com/apache/incubator-hugegraph-ai/blob/main/hugegraph-llm/src/hugegraph_llm/config/config_data.py#L65-L70. However, you're right that using `""`  would be more secure.

Reply:
> I'm using `"admin"` here because the default config in [`main`/hugegraph-llm/src/hugegraph_llm/config/config_data.py#L65-L70](https://github.com/apache/incubator-hugegraph-ai/blob/main/hugegraph-llm/src/hugegraph_llm/config/config_data.py?rgh-link-date=2024-09-22T17%3A25%3A06Z#L65-L70). However, you're right that using `""` would be more secure.

That's to display a default value on the **UI**, but in reality, using `xxx` is more appropriate (for user understanding), and a default value is used for **non-UI** scenarios

Reply:
I understand, you're right.